### PR TITLE
enclave-tls/dist: add the postun rules in the rpm spec

### PR DESCRIPTION
**Pull request by hustliyilin**
Signed-off-by: Yilin Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

**Review by haosanzi — APPROVED** [review]

File: enclave-tls/dist/rpm/enclave_tls.spec [hunk 1]
```diff
@@ -51,9 +54,12 @@ popd
 
 %install
 pushd %{name}
-Enclave_Tls_Root=%{?buildroot}/opt/enclave-tls Enclave_Tls_Bindir=%{?buildroot}/usr/share/enclave-tls/samples make install
+Enclave_Tls_Root=%{?buildroot}%{ENCLAVE_TLS_ROOTDIR} Enclave_Tls_Bindir=%{?buildroot}%{ENCLAVE_TLS_BINDIR} make install
 popd
 
+%postun
+rm -rf %{ENCLAVE_TLS_ROOTDIR} %{ENCLAVE_TLS_BINDIR}
```
Comment (jiazhang0) [inline enclave-tls/dist/rpm/enclave_tls.spec:61]: /usr/share/enclave-tls/ should be removed as well.

Comment (hustliyilin) [reply enclave-tls/dist/rpm/enclave_tls.spec]: OK

File: enclave-tls/dist/rpm/enclave_tls.spec [hunk 2]
```diff
@@ -51,9 +54,12 @@ popd
 
 %install
 pushd %{name}
-Enclave_Tls_Root=%{?buildroot}/opt/enclave-tls Enclave_Tls_Bindir=%{?buildroot}/usr/share/enclave-tls/samples make install
+Enclave_Tls_Root=%{?buildroot}%{ENCLAVE_TLS_ROOTDIR} Enclave_Tls_Bindir=%{?buildroot}%{ENCLAVE_TLS_BINDIR} make install
 popd
 
+%postun
+rm -rf %{ENCLAVE_TLS_ROOTDIR} /usr/share/enclave-tls
```
Comment (jiazhang0) [inline enclave-tls/dist/rpm/enclave_tls.spec:61]: Use `$(dirname %{ENCLAVE_TLS_ROOTDIR})`

Comment (hustliyilin) [reply enclave-tls/dist/rpm/enclave_tls.spec]: > Use `$(dirname %{ENCLAVE_TLS_ROOTDIR})`

Done, thanks.

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**